modules/moving-the-gitops-operator-pod-to-infrastructure-nodes.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assembly:
+//
+// * gitops_workloads_infranodes/running-gitops-control-plane-workloads-on-infrastructure-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="moving-the-gitops-operator-pod-to-infrastructure-nodes_{context}"]
+= Moving the {gitops-shortname} Operator pod to infrastructure nodes
+
+You can move the {gitops-shortname} Operator pod to the infrastructure nodes.
+
+.Prerequisites
+* You have installed the {gitops-title} Operator in your cluster.
+* You have access to the cluster with `cluster-admin` privileges.
+
+.Procedure
+
+. Label an existing node as infrastructure node by running the following command:
++
+[source,terminal]
+----
+$ oc label node <node_name> node-role.kubernetes.io/infra=
+----
++
+.Example output
+[source,terminal]
+----
+node/<node_name> labeled
```

Comment:
mention the node name since this is the example output

Reply:
also, [source,text]

Reply:
In GitOps docs, we use <> again in the example if the previous command used that without a specific example. So I will keep it for the docs consistency.

Also I have never seen [source,text] instead [source,terminal] for these. In the guidelins we have this line: "If a message is too long to include inline, put it inside a code block with [source,text] metadata. " and "[source,terminal]" for example output.. Here:https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#node-names so I will keep it as  [source,terminal] as [source,text] does not seem to be right here. Thank you!